ui/design-system-components/src/components/Entry/EntryCard/index.tsx

```diff
@@ -146,13 +146,11 @@ const EntryCard: React.FC<EntryCardProps> = props => {
             </Tooltip>
           )}
           {entryData?.createdAt &&
-            (!editable && notEditableLabel ? (
+            (editable ? (
               <Tooltip placement={'top'} content={notEditableLabel}>
                 {editableIconButtonUi}
               </Tooltip>
-            ) : (
-              editableIconButtonUi
-            ))}
+            ) : null)}
```

Comment:
That logic was there for a reason :) Can you share the reason why removed it ?

The logic was there for two reasons. One, when `notEditableLabel` is set then it adds unnecessary tooltip element into a page and two when you hover on the `edit` icon then you will see unwanted arrow.

![image](https://github.com/AKASHAorg/akasha-core/assets/35398733/c7f7e2bc-5a33-46ac-b33d-1931e1d723c4)

Reply:
thanks for pointing it out Didd :). I overlooked the notEditableLabel prop. Will take a closer look.